pymystrom/__init__.py

```diff
@@ -61,3 +63,59 @@ async def _request(
         return response_json
 
     return response.text
+
+
+class MyStromDeviceType(IntEnum):
+     SWITCH_CH_V1 = 101
+     BULB = 102
+     BUTTON_PLUS = 103
+     BUTTON = 104
+     LED_STRIP = 105
+     SWITCH_CH_V2 = 106
+     SWITCH_EU = 107
+     MOTION_SENSOR = 110
+     MODULO = 113
+     BUTTON_PLUS_2ND = 118
+     SWITCH_ZERO = 120
```

Comment:
The mapping is already [available](https://github.com/home-assistant-ecosystem/python-mystrom/blob/master/pymystrom/discovery.py#L8).

Reply:
I removed the enum. 

It might me helpful in a subsequent PR to merge the 2 approaches (enum and dict): The adventage of the enum is that the python interpreter checks the "label" and reduces the risk of typos. What do you think!